src/components/BookAdd.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+function BookAdd() {
+  return (
+    <>
+      <h2>ADD NEW BOOK</h2>
+      <input type="text" placeholder="Book Title .." />
+      <input type="text" placeholder="Book Author .." />
+      <form action="#">
+        <select id="books" name="books">
+          <option value="Fiction">SciFiction</option>
+          <option value="Action">Action</option>
+          <option value="Adventure">Adventure</option>
+          <option value="Romance">Romance</option>
+        </select>
+      </form>
+      <button type="button">ADD BOOK</button>
+    </>
+  );
+}
+
+export default BookAdd;
```

Comment:
Since you're not adding books for now, but sooner you'll be adding a book to the bookstore API and for that, you have to have a form with input fields that is get submitted with the data captured from the form.  Your form needs to have all the input fields within it and a submit button which you can set its type to submit so that when it's clicked, it submits the form(for later implementation). 

- [ ] Kindly re-arrange the two input fields above the form tag to go inside the form and the button below the closing form tag to be inside the form in order for it to submit your data when it's clicked. See the direction from the screenshot below marked in red arrows 👍 

![image](https://user-images.githubusercontent.com/50941074/199941164-441c096a-2373-4377-b51f-0827e15b745b.png)

Reply:
That's true. Thanks for that spot.